### PR TITLE
Add "I want to contribute" badge to README

### LINKED ISSUE #818: Design an "I want to contribute!" badge for the readme
### Summary

I found this "I want to contribute!" button on the BinderHub readme. It links to the contributing guidelines and I think it's a cool way of directing people to the relevant information to getting started.

I want to contribute button in BinderHub readme

### What needs to be done?

- [ ] Design a _Turing Way_ version

### Who can help?

* Anyone

---

### PR DESCRIPTION
**Pull request by erictleung**
### Summary

Fixes #818

### List of changes proposed in this PR (pull-request)

* Add contributor badge to README file

### What should a reviewer concentrate their feedback on?

- [x] Is the text okay?
- [x] Is the color okay?
- [x] Everything looks ok?

### Acknowledging contributors

- [X] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by erictleung** [comment]
> 
> 
> Thanks @erictleung! This looks great. Is it ok to change the color as the newsletter uses the current color already?

Thanks for the feedback. And no problem! I went with that color to be consistent with the branding. 

Take a look at the colors here and let me know what works https://github.com/alan-turing-institute/the-turing-way/issues/818#issuecomment-583535506. You *may* be able to use a custom hex color code as well, but I haven't tried.

**Comment by malvikasharan** [comment]
The newsletter is from tinyurl (external resource as well). Can use yellow/yellow-green? No specific reason, but this is the only colour not used in another contexts.

**Comment by erictleung** [comment]
> 
> 
> The newsletter is from [tinyletter](https://tinyletter.com/TuringWay/archive) (external resource as well). Can we use yellow/yellow-green? No specific reason, but this is the only colour not used in another context.

Ah makes sense. Well here are those two colors.

**Yellow**

[](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)

**Yellow-green**

[](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)

My vote is for yellow. Let me know if that's fine.

**Comment by malvikasharan** [comment]
Yellow looks good to me too :) Approving this PR. Thank you so much for such a thoughtful contribution @erictleung! 🙌

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
...
+[![](https://img.shields.io/static/v1?label=TuringWay&message=I%20want%20to%20contribute!&color=blueviolet&logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8%2F9hAAACYklEQVQ4jXXTy09TQRTH8f5VPhI1xoVxYURNAFcmRleaGDdGXQlKAYkLUARNfICoScGKpTyE3t5bkKD2AUQepUXB0gcgLTalD9rema8LKRVrT3I2k%2Fl95kwyY6BMfQiFqHaoVDlUBoJBZJl9hn8XRsIhqh0abd55tnWdrBA8WfBSpakMhUqhXUCJhKl2aLR65%2FEtLeGc%2BYoy5aHf46bX7cThctK%2BAw2HQkVAW41wzqHRMjNNRteR%2BQzGjg5udZtQ47FiO50gdLZ1nVbvPNUOFSUSxnB4sJ%2F0TjCTTjHk%2BoJl%2BRtqPEaL6zMH79Rw0dyDVVURqRgyn0EkN8jkshwZGsBQodgQyQ2kyDPsce859drjdqLRKE0D%2FZhHR5F6DpHc2B3%2FjF3BcFqxARIpBXXmt9ii67vAYDhIr8fNx0UfE3OzzC0sIHIpxNYqSPEHqFBsiFQMkU3h8vs5%2FvABTeNje6BCj%2FxcwzLlIZHYROq5v4EoIr2JyCbJ57Kobjd3u7o41v4I68pyCfTGrhSvUKHYAJD5bcTWGjKbJJdO4A8E6JyexP4rWgK8Vkb2AjK7hcxnmZybxfF9kff%2BhZJQofvXwhg7O4vAfU2l79ME79xOrjY3c9ZYVzZs8nvZf6%2BRQCRCTgiODg1iCK6vc6WtjZM1tzlRW8sNa99%2Fx64fH%2BNAQz0un49nfh%2BVmspAcKX4lKWUbMbjXOg2cf3Vy%2BLIoRWqekxc7nhB6%2FQ0lZqKJRBAyjKfKZFIcKixgVPPn3LTamFfUyPne7qp1Oz0Bn4g5d7vVAIUamJ2FqPZzCW7gvlHabBQvwE2XnlAiFRrOwAAAABJRU5ErkJggg%3D%3D)](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)
```
Comment (malvikasharan) [inline README.md:7]:
```suggestion
[![](https://img.shields.io/static/v1?label=TuringWay&message=I%20want%20to%20contribute!&color=yellow&logo=data%3Aimage%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8%2F9hAAACYklEQVQ4jXXTy09TQRTH8f5VPhI1xoVxYURNAFcmRleaGDdGXQlKAYkLUARNfICoScGKpTyE3t5bkKD2AUQepUXB0gcgLTalD9rema8LKRVrT3I2k%2Fl95kwyY6BMfQiFqHaoVDlUBoJBZJl9hn8XRsIhqh0abd55tnWdrBA8WfBSpakMhUqhXUCJhKl2aLR65%2FEtLeGc%2BYoy5aHf46bX7cThctK%2BAw2HQkVAW41wzqHRMjNNRteR%2BQzGjg5udZtQ47FiO50gdLZ1nVbvPNUOFSUSxnB4sJ%2F0TjCTTjHk%2BoJl%2BRtqPEaL6zMH79Rw0dyDVVURqRgyn0EkN8jkshwZGsBQodgQyQ2kyDPsce859drjdqLRKE0D%2FZhHR5F6DpHc2B3%2FjF3BcFqxARIpBXXmt9ii67vAYDhIr8fNx0UfE3OzzC0sIHIpxNYqSPEHqFBsiFQMkU3h8vs5%2FvABTeNje6BCj%2FxcwzLlIZHYROq5v4EoIr2JyCbJ57Kobjd3u7o41v4I68pyCfTGrhSvUKHYAJD5bcTWGjKbJJdO4A8E6JyexP4rWgK8Vkb2AjK7hcxnmZybxfF9kff%2BhZJQofvXwhg7O4vAfU2l79ME79xOrjY3c9ZYVzZs8nvZf6%2BRQCRCTgiODg1iCK6vc6WtjZM1tzlRW8sNa99%2Fx64fH%2BNAQz0un49nfh%2BVmspAcKX4lKWUbMbjXOg2cf3Vy%2BLIoRWqekxc7nhB6%2FQ0lZqKJRBAyjKfKZFIcKixgVPPn3LTamFfUyPne7qp1Oz0Bn4g5d7vVAIUamJ2FqPZzCW7gvlHabBQvwE2XnlAiFRrOwAAAABJRU5ErkJggg%3D%3D)](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)
```

**Pull request merged**